### DISCUSSION TITLE
Help Installing v10 version of lib

### DISCUSSION THREAD

**Discussion by kevinmanncito** [rnmapbox/maps#2007] [Q&A] [answered]
I'm sure I'm just missing something obvious but can't seem to figure out what is happening here. Has anyone else seen this issue? Trying to run `yarn add rnmapbox/maps#main` or `yarn add https://github.com/rnmapbox/maps#main` and I'm getting the error:

```
zsh: no matches found: https://github.com/rnmapbox/maps#main
```

<img width="588" alt="Screen Shot 2022-06-28 at 4 39 05 PM" src="https://user-images.githubusercontent.com/2521298/176316312-ca8c9db7-3d40-4e0c-9d90-afe0307556f7.png">

Anyone else seen this before? Thanks in advance 🙏

**Comment by mfazekas** [comment]
Can you run these as well:
```
which yarn
yarn --version
```

**Reply by kevinmanncito** [reply]
`which yarn`

```
/usr/local/bin/yarn
```

`yarn --version`

```
1.22.17
```
<img width="544" alt="Screen Shot 2022-06-29 at 8 54 44 AM" src="https://user-images.githubusercontent.com/2521298/176468175-e084493a-cb9d-4fac-bf12-6c1ec6e67757.png">

**Comment by mfazekas** [comment, marked as answer]
Strange, normally 
```
zsh:rnocmatches found: rnmapbox/maps#main
``` 
means that zsh treated `rnmapbox/maps#main`  a pattern. But not sure why `#` would be a pattern in your zsh.


Normally quote solves this issue, but not sure why it's showing up on your zsh first place, adding quotes might work:
```
echo what?
zsh: no matches found: what?
echo 'what?'
what?
```

**Reply by kevinmanncito** [reply]
That did the trick, thanks!

```
yarn add "rnmapbox/maps#main"
```
